syntax = "proto3";

package RocketryProto;

message ServoInit {
  uint32 pin = 1;
  int32 safePosition = 2;
}

message ServoControl {
  uint32 pin = 1;
  int32 position = 2;
}

message ArduinoIn {
  oneof data {
    ServoInit servoInit = 1;
    ServoControl servoControl = 2;
  }
}